pkg/controller/application_spec_sync.go

```diff
@@ -0,0 +1,50 @@
+package controller
+
+import (
+	"fmt"
+
+	"github.com/jackc/pgx/v4/pgxpool"
+	"k8s.io/apimachinery/pkg/api/equality"
+	appsv1 "sigs.k8s.io/application/api/v1beta1"
```

Comment:
this alias may be confusing.
should be `appsv1beta1` (v1beta1 is the version.. beta makes it clear it's still not production level package)